app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self._health = health
+        self.hidden = hidden
+
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return (
+            "{"
+            f"Name: {self.name}, "
+            f"Health: {self.health}, "
```

Comment:
The `health` property should be accessed using `self.health` instead of `self._health` to ensure the getter method is used.